### ISSUE TITLE
Badges: One badge for a subset of repositories

### ISSUE THREAD

**Issue by Profpatsch** [repology/repology#762]
For https://salsa.debian.org/reproducible-builds/reproducible-website we want to add a list of package versions for every major Distro. The big badge is a bit too big to be useful, so it would be nice to have a filtered badge. Something like:

```
https://repology.org/badge/vertical-allrepos/diffoscope.svg?repositories=arch,debian,nix_stable,
```

**Comment by luzpaz** [comment]
I think this feature will be an incentive for more integration of repology badges in the github/gitlab/bitbucket/ ecosystem. Several repos that I've sent PRs to which adds a repology badge to their README have an issue with the length of the badge displaying non-relevant distros like Rosa or pisilinux etc... 

This feature allows for adding the most mainstream repos/distros that folks recognize.

**Comment by AMDmi3** [comment]
This is doable. We need
- [ ] distro aggregation first (#514)
- [ ] then an aggregated badge type
- [ ] then an additional filtering option

**Comment by gpotter2** [comment]
That would indeed be a very cool feature.

I wanted to use Repology on scapy's [README](https://github.com/secdev/scapy) and [website](https://scapy.net/), but can't find a way for it not to take half of the screen with distros we don't support.

**Comment by AMDmi3** [comment]
> but can't find a way for it not to take half of the screen

Note that you can make it floating so it doesn't waste the vertical space regardless of how long it is. See
https://github.com/radare/radare2 for example. This is mentioned on badges page as well.

> distros we don't support. We would only need

The fact you're going to discriminate distros makes me hesitant to add filtering.

**Comment by luzpaz** [comment]
If I may add my 2cents, I've encountered some resistance from devs to add the badge  of several FOSS projects for the reason above (floating or not) because they don't see the utility in showing availability of their packages in repos like AIX or Rosa etc... IMO this is bottlenecking adoption of a very needed service that repology provides.

**Comment by AMDmi3** [comment]
"Adoption" is not in a list of Repology goals. The goal is to provide complete and correct packaging information.

**Comment by luzpaz** [comment]
Agreed, well it seems to me that two are mutually exclusive. This is feedback from project devs who could be more interested in correct packaging if they were made aware based on their ranking in repology etc...

**Comment by gpotter2** [comment]
> The fact you're going to discriminate distros makes me hesitant to add filtering.

I didn’t intend to discriminate any distro, the real argument is about screen space. I knew of `align=right` (yes I have read the docs :-) ) before, but it still take to much space in most cases. It’s is not offensive to rationally select the most used distros to be displayed first.

Also, the link is still clickable: there could be a `see more [click here]` tag at its bottom, linking to the repology webpage, where the full list is available.

**Comment by AMDmi3** [comment]
Well let's start from grouping and see how it goes.

**Comment by gpotter2** [comment]
Sorry if my messages seem harsh, It’s really not wanted. Repology is a great project and I respect fully your points. :)
